awx/main/access.py

```diff
@@ -2627,7 +2627,7 @@ def filtered_queryset(self):
         return self.model.objects.filter(
             Q(organization__in=Organization.accessible_pk_qs(self.user, 'read_role'))
             | Q(unifiedjobtemplate_labels__in=UnifiedJobTemplate.accessible_pk_qs(self.user, 'read_role'))
-        )
+        ).distinct()
```

Comment:
the second Q filter involves a Many To Many relationship, which has the possibility of returning duplicate label entries. We can call `distinct()` to get unique labels.